app/src/main/java/org/sopt/and/ui/login/LogInScreen.kt

```diff
@@ -36,69 +38,58 @@ import androidx.compose.runtime.rememberCoroutineScope
 import androidx.compose.runtime.setValue
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
-import androidx.compose.ui.input.key.Key.Companion.I
+import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.platform.LocalContext
 import androidx.compose.ui.res.colorResource
+import androidx.compose.ui.res.painterResource
 import androidx.compose.ui.res.stringResource
-import androidx.compose.ui.text.font.FontWeight
 import androidx.compose.ui.text.input.ImeAction
-import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.unit.dp
 import androidx.compose.ui.unit.sp
+import androidx.lifecycle.compose.collectAsStateWithLifecycle
+import androidx.lifecycle.viewmodel.compose.viewModel
+import com.google.accompanist.permissions.ExperimentalPermissionsApi
 import kotlinx.coroutines.launch
-import org.sopt.and.MyActivity
 import org.sopt.and.R
-import org.sopt.and.SignUpActivity
-import org.sopt.and.component.LogInTextField
+import org.sopt.and.ui.component.findActivity
+import org.sopt.and.ui.signup.SignUpState
+import org.sopt.and.ui.signup.SignUpViewModel.Companion.EXTRA_SIGNUP_IMAGE_LIST
 
 
-val imageList = listOf(
-    Icons.Default.CheckCircle,
-    Icons.Default.CheckCircle,
-    Icons.Default.CheckCircle,
-    Icons.Default.CheckCircle,
-    Icons.Default.CheckCircle
-)
-
+@ExperimentalPermissionsApi
 @Composable
 fun LogInScreen(
-    modifier: Modifier = Modifier
+    navigateToSignUp: () -> Unit,
+    navigateToMyPage: (String, String) -> Unit,
+    signUpState: SignUpState
 ) {
+    val viewModel = viewModel<LogInViewModel>()
 
     val context = LocalContext.current
+    val activity = context.findActivity()
 
-    var id by remember { mutableStateOf("") }
-    var password by remember { mutableStateOf("") }
-
-    var signUpId by remember { mutableStateOf("") }
-    var signUpPassword by remember { mutableStateOf("") }
-    var showPassword by remember { mutableStateOf(false) }
+    val loginState = viewModel.loginState.collectAsStateWithLifecycle()
```

Comment:
```suggestion
    val loginState by viewModel.loginState.collectAsStateWithLifecycle()
```

로 쓰면 아래에서는 ``.value`` 굳이 호출 안하셔도 됩니다.

Reply:
.value가 항상 거슬렸는데 감사합니다